Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -157,6 +158,13 @@ class BoolSetting: Setting {
         control.addTarget(self, action: "switchValueChanged:", forControlEvents: UIControlEvents.ValueChanged)
         control.on = prefs.boolForKey(prefKey) ?? defaultValue
         cell.accessoryView = control
+        cell.detailTextLabel?.snp_makeConstraints { make in
```

Comment:
After thinking about this I don't see another easier way of doing the text wrapping without creating a subclass so I think we'll be OK. I would clean this up a bit though by extracting the if outside of the `snp_makeConstraints` call and pinning to the right of the title so avoid magic numbers:

```swift
if let titleLabel = cell.textLabel {
    cell.detailTextLabel?.snp_makeConstraints { make in
        make.left.right.equalTo(title)
        make.top.equalTo(title.snp_bottom)
    }
}

```